.travis.yml

```diff
@@ -13,8 +13,10 @@ python:
   - 3.7
   - 3.8
 
+# Testing 1.0.3 as it is still pre-Celery, but has the choices test fix nautobot#457
 env:
-  - "INVOKE_NAUTOBOT_CHATOPS_NAUTOBOT_VER=1.0.2"
+  - "INVOKE_NAUTOBOT_CHATOPS_NAUTOBOT_VER=1.0.3"
```

Comment:
So is the issue here that the plugin works fine with 1.0.2 but tests will fail with that version? Might be good to state that in the README or similar then.

Reply:
They don't actually fail in 1.0.2 either, it just doesn't do the choices tests.  The Choices test was added in 1.0.3 and failing when I ran 1.1.2 because I was missing `choices_fields`